force-app/main/default/lwc/reviewSessions/reviewSessions.js

```diff
@@ -291,4 +305,38 @@ export default class ReviewSessions extends LightningElement {
         this.isLoaded = true;
         this.dispatchEvent(new CustomEvent("loaded", { detail: this.isLoaded }));
     }
+
+    handleStartChange(event) {
+        if (!event.detail.value) {
```

Comment:
I like that this check is added. But would there ever be a scenario where this value is null or undefined and we have to null out the end date.

Reply:
Yes, if the user clears out the start date all together then we want the end date to get cleared out as well. This behavior is the same on the new service schedule first session start and end date but looking at the code again I do see a type, it shouldn't have .value so I will fix that now. Thanks!